exercises/hangman/.meta/src/reference/java/Output.java

```diff
@@ -0,0 +1,39 @@
+import java.util.List;
+import java.util.Set;
+
+class Output {
+
+    public final String secret;
+    public final String discovered;
+    public final Set<String> guess;
+    public final Set<String> misses;
+    public final List<Part> parts;
```

Comment:
In reactive programming, we want to avoid side effects as much as possible. This would include having a mutable state - which is represented by the Output class - as we don't know who and when will consume such outputs.
So for this exercise, I wanted an immutable state. And you are right that I shall make sure that it is truly immutable.

Reply:
I might have missed something, but it looks like this is still mutable to me.

Reply:
Nope, that's true. I missed this one.